Formula/grpc.rb

```diff
@@ -0,0 +1,44 @@
+class Grpc < Formula
+  desc "Google's open-source RPC framework, based on protocol buffers."
+  homepage "http://www.grpc.io/"
+  head "https://github.com/grpc/grpc.git"
+
+  stable do
```

Comment:
Hmm, I don't think we currently have a way to specify stable-only build dependencies, unfortunately. 

I guess in that case a `stable` block is fine, since there's no point in pulling in `gnu-sed` for head builds.